lib/resources/r4/encounter.yaml

```diff
@@ -528,6 +528,22 @@ fields:
       }
     }
 
+- name: location
+  required: 'No'
+  type: Reference (Location)
+  description: The location where the encounter takes place.
+  action: create
+  example: |
+    {
+      "location": [
+        {
+          "location": {
+            "reference": "Location/633869"
+          }
+        }
+      ]
+    }
+
```

Comment:
This is where body field notes should go

Reply:
yup added here https://github.com/cerner/fhir.cerner.com/pull/803/commits/1d0f2634bd131127317a63f5458e2f27fc579829#diff-ea755c3df7ed8869a54a274cedf2e40e119b2f16a6e3ed8bc8ae211e0e87389cL545